OpenRA.Game/Game.cs

```diff
@@ -939,5 +939,10 @@ public static void FinishBenchmark()
 				Exit();
 			}
 		}
+
+		public static void Clean()
```

Comment:
Can we call this `ClearDelayedActions`?

Reply:
This actually raises a bigger issue: why is the server queuing delayed actions in the first place, knowing that will never be run (`LogicTick` isn't called)?

An even simpler fix could be to add an if-not-server check to the thing in the map cache that is queueing these useless tasks.

Reply:
Something like 7bbaa4378008741821e496179fe82ee86a67a769 to consume the events immediately would IMO be the best compromise between simplicity/cleanness.